site/src/component/GradeDist/Pie.tsx

```diff
@@ -153,20 +143,15 @@ export default class Pie extends React.Component<PieProps> {
   }
 
   styleTooltip = (props: PieTooltipProps<Slice>) => {
-    return (
-      <div style={tooltipStyle.tooltip?.container}>
-        <strong>
-          {props.datum.id}: {((props.datum.value / this.total) * 100).toFixed(2)}%
-        </strong>
-      </div>
-    );
+    return <ChartTooltip label={props.datum.id} value={((props.datum.value / this.total) * 100).toFixed(2) + '%'} />;
   };
 
   render() {
+    const data = this.getClassData();
```

Comment:
what is the actual class data? Assuming it's a grade distribution, we should make these variables names more clear:

`const gradeCounts = ...`
`gradeCounts.map(grade => grade.color)`